tests/Unit/Utilities/CssConcatenator/RuleBlockTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Tests\Unit\Utilities\CssConcatenator;
+
+use Pelago\Emogrifier\Utilities\CssConcatenator\RuleBlock;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test case.
+ *
+ * @author Oliver Klee <github@oliverklee.de>
+ *
+ * @covers \Pelago\Emogrifier\Utilities\CssConcatenator\RuleBlock
```

Comment:
Note: I've already switched positions of the `@author` and `@covers` annotations in my local copy.